app/knowledge_panels.py

```diff
@@ -39,3 +41,59 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(
+    facet: str,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "packaging":
+        plural = facet
+    else:
+        plural = facet_plural(facet=facet)
+    query = {
+        "fields": "code,last_editor,last_modified_t",
+        "sort_by": "last_modified_t",
+    }
+    if facet == "country":
+        country_code = country_to_ISO_code(value=value)
+        country = value
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        description = f"{value}"
+    else:
+        if value is not None:
+            url = "https://world.openfoodfacts.org"
+            query[f"{plural}_tags_en"] = value
+            description = f"{value} based for {facet}"
+        if country is not None:
+            country_code = country_to_ISO_code(value=country)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+            description = f"{facet} based for {country}"
+
+    search_url = f"{url}/api/v2/search"
+    description = f"last-edits issues related to {description}"
+    response_API = requests.get(search_url, params=query)
+    data = response_API.json()
+    counts = data["count"]
+    tags = data["products"]
```

Comment:
optional: I think it would be better if we isolate this in a function, maybe in a "off.py" file where we put functions calling the openfoodfacts API.

(only the line for description should be kept here :-) )